variables.tf

```diff
@@ -183,3 +182,34 @@ variable "disable_mfa" {
   type        = bool
   default     = false
 }
+
+variable "disabled" {
+  description = "Specifies whether the user is disabled, which prevents logging in and aborts all the currently-running queries for the user."
+  type        = bool
+  default     = false
+}
+
+variable "naming_scheme" {
```

Comment:
what if we change this to `name_decorator`?
we would have usage like:
```
name = "foo"
name_decorator = {}
```

Reply:
It was decided to go with `name_scheme`